examples/blockly-react/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "dependencies": {
     "@blockly/field-date": "^8.0.0",
-    "blockly": "^10.0.0",
```

Comment:
Did you want to commit these changes? Slightly concerned that only updating a few of them will mean we miss updating them to the actual release when we do that. We've definitely missed things doing find-and-replace because of stuff like this.

Reply:
@maribethb said she usually searches for 'beta' to make sure she updates everything; I don't have a strong opinion, since I think the updated imports will work in either case.

Reply:
Since they should work in v10 as well, it's fine either way imo. We definitely have missed some in the past but I do search the term beta when I do the rc branch merge so honestly I'm more likely to find this than if it were on some random 10.x version